### PR TITLE
Feature/17

### LINKED ISSUE #17: ReadMe file lacks instructions for postgres
The readme file does not have instructions to create development database and to edit the corresponding file to append their passwords for proper firing of rails server

### PR DESCRIPTION
**Pull request by KrzysztofMadejski**
Resolves #17 ReadMe file lacks instructions for postgres:
- bash commands to install dev & test database and users
- rename `database.yml` to `database.yml.example` to allows devs to have custom config
- update README

Ping @daas-ankur-shukla @Mr0grog

### CODE REVIEW

**Comment by KrzysztofMadejski** [comment]
Will see if anything should be added to any deployments scripts that potentially can utilise old `database.yml`...

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,4 @@
 # Configuration files
 .powenv
 .env
+config/database.yml
```
Comment (Mr0grog) [inline .gitignore]: As noted back on [#17 (more reasoning there)](https://github.com/edgi-govdata-archiving/web-monitoring-db/issues/17#issuecomment-307125039), I don’t think we should do this.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,10 @@ It’s a Rails app that:
 
 ## Installation
 
-1. Ensure you have Ruby 2.4.1+
+1. Ensure you have Ruby 2.4.1+. 
+    
+    You can user [rbenv](https://github.com/rbenv/rbenv) to manage multiple Ruby versions
```
Comment (Mr0grog) [inline README.md]: `user` → `use`

File: README.md [hunk 3]
```diff
@@ -40,15 +43,42 @@ It’s a Rails app that:
...
+7. Set up your database
 
+    Make a local copy of the database configuration template file:
+    
     ```sh
-    $ bundle exec rake db:setup
+    cp config/database.yml.example config/database.yml
+    ```
+
+    First create users, databases and configure extensions:
```
Comment (Mr0grog) [inline README.md]: I don’t think this should be recommended as a default configuration. In the vast majority of cases, there’s no reason anyone needs an extra password on their dev database. It’s great that we can support that, but it’s not necessary and we shouldn’t burden basic setup with a lot of unnecessary steps.

If a user wants to password-protect their local DB, we should cover that in the “advanced setup” section.

File: README.md [hunk 4]
```diff
@@ -40,15 +43,42 @@ It’s a Rails app that:
...
+
+    ```sh
+    sudo -u postgres psql -c "CREATE USER wmdb_dev WITH PASSWORD 'wmdb_dev';"
+    sudo -u postgres createdb -O wmdb_dev wmdb_dev -E utf-8
+    sudo -u postgres psql -c 'CREATE EXTENSION IF NOT EXISTS "uuid-ossp";' wmdb_dev
+    sudo -u postgres psql -c 'CREATE EXTENSION IF NOT EXISTS "plpgsql";' wmdb_dev
```
Comment (Mr0grog) [inline README.md]: Adding these two extensions is already done by `rake db:schema:load`; no need to specify it manually.

File: README.md [hunk 5]
```diff
@@ -40,15 +43,42 @@ It’s a Rails app that:
...
     ```sh
-    $ bundle exec rake db:setup
+    cp config/database.yml.example config/database.yml
+    ```
+
+    First create users, databases and configure extensions:
+
+    ```sh
+    sudo -u postgres psql -c "CREATE USER wmdb_dev WITH PASSWORD 'wmdb_dev';"
+    sudo -u postgres createdb -O wmdb_dev wmdb_dev -E utf-8
```
Comment (Mr0grog) [inline README.md]: `rake db:create` (or `rake db:setup`) should use the username and password specified in `DATABASE_URL` or `database.yml`. I don’t think this line is necessary.

**Comment by KrzysztofMadejski** [comment]
I'm closing that, will reopen a new one.

File: README.md (hunk 4 above)
Comment (KrzysztofMadejski) [inline README.md]: If your postgres user is not a superuser then it won't have rights to create extensions through schema load and you need to setup them via `postgres` superuser in this way.

Comment (Mr0grog) [inline README.md]: Ah, fair point. Same for the `createdb` line, I suppose.

Comment (KrzysztofMadejski) [inline README.md]: That's right

**Pull request closed without merging**